frontend/src/components/AgentStatusBar.tsx

```diff
@@ -88,16 +90,25 @@ function AgentStatusBar() {
     ) {
       if (document.cookie.indexOf("audio") !== -1) beep();
     }
-  }, [curAgentState]);
+  }, [curAgentState, dispatch]);
```

Comment:
`dispatch` is not being used within the `useEffect` and it is unlikely the function definition would change to cause the effect to trigger. Was this added intentionally?

Reply:
Good catch, it's a remnant from earlier commits, which I'll remove with the next one. Thanks!

Reply:
Done!